### ISSUE TITLE
Associate 2016 daubenmire grass data with 2016 native status

### ISSUE THREAD

**Issue by sethapp** [jarad/ISUmonarch#69]
In 2016 I knew what the native status was of the grasses in the plots. (this is prior to our team modifying the existing landscape). I want the 'csg' and 'wsg' from 2016 daubenmire data to be associated with the '2016_csg_native_status' and the '2016_wsg_native_status' from this file which will eventually be a data.frame https://github.com/jarad/ISUmonarch/blob/master/data-raw/transect/site_details.csv. 

For example; in 2016 we are able to say a site had 38% cover of csg **and** that the csg was introduced species. 

I hope this explanation makes sense.

**Comment by nulloa** [comment]
Ok so this should do what you want:

```
library(ISUmonarch)
library(tidyverse)

head(cover)
head(site)

site16 <- site %>%
  select(siteID, transectID, `2016_csg_native_status`, `2016_wsg_native_status`) %>%
  gather("class","status",-siteID, -transectID) %>%
  mutate(class = sapply(strsplit(site16$class,"_"), `[`, 2))
head(site16)

cover %>%
  left_join(site16, by=c("siteID", "transectID", "class")) %>%
  head()
```

which gives you this:
```
> cover %>%
+   left_join(site16, by=c("siteID", "transectID", "class")) %>%
+   head()
  year month day siteID transectID round section                   observer class percentage     status
1 2016     6  27   nie1     tnie1a     1      0m JaclynAKelseyFSydneyHCoryH   csg         38 introduced
2 2016     6  27   nie1     tnie1a     1     10m JaclynAKelseyFSydneyHCoryH   csg         38 introduced
3 2016     6  27   nie1     tnie1a     1     20m JaclynAKelseyFSydneyHCoryH   csg         63 introduced
4 2016     6  27   nie1     tnie1a     1     30m JaclynAKelseyFSydneyHCoryH   csg         38 introduced
5 2016     6  27   nie1     tnie1a     1     40m JaclynAKelseyFSydneyHCoryH   csg         38 introduced
6 2016     6  27   dav1     tdav1a     1      0m             JaclynASydneyH   csg         38 introduced
```

I can make this into a vignette though I'm not sure if it's general enough to do that, but it may be useful for the future.

**Comment by nulloa** [comment]
Went ahead and made it into a vignette: 1c744ba9b4dadf856472b04b1f819ce53747ec57

**Comment by nulloa** [comment]
Closed with 1c744ba9b4dadf856472b04b1f819ce53747ec57
